src/main/java/net/dv8tion/jda/internal/entities/RoleImpl.java

```diff
@@ -546,7 +582,10 @@ public String toString()
             return new EntityString(this)
                     .addMetadata("bot", getBotId())
                     .addMetadata("integration", getIntegrationId())
+                    .addMetadata("subscriptionListing", getSubscriptionId())
                     .addMetadata("isBoost", isBoost())
+                    .addMetadata("isAvailableForPurchase", isAvailableForPurchase())
+                    .addMetadata("isGuildConnections", isGuildConnections())
                     .toString();
```

Comment:
We should include a call to `.setName(this.getName())`

Reply:
There is no name for role tags

Reply:
Oh, this is RoleTag, not Role. my b